model/src/w3gridmd.F90

```diff
@@ -995,10 +998,13 @@ MODULE W3GRIDMD
 #endif
 #ifdef W3_NL1
   NAMELIST /SNL1/ LAMBDA, NLPROP, KDCONV, KDMIN,                  &
-       SNLCS1, SNLCS2, SNLCS3
+       SNLCS1, SNLCS2, SNLCS3,                         &
+       IQTYPE, TAILNL, GQMNF1, GQMNT1,                 &
+       GQMNQ_OM2, GQMTHRSAT, GQMTHRCOU, GQAMP1, GQAMP2, GQAMP3, GQAMP4
 #endif
 #ifdef W3_NL2
-  NAMELIST /SNL2/ IQTYPE, TAILNL, NDEPTH
+  NAMELIST /SNL2/ IQTYPE, TAILNL, NDEPTH, GQMNF1, GQMNT1,         &
+       GQMNQ_OM2, GQMTHRSAT, GQMTHRCOU, GQAMP1, GQAMP2, GQAMP3, GQAMP4
```

Comment:
@mickaelaccensi can you add these new namelist to model/inp/ww3_grid.inp?

Reply:
@mickaelaccensi  this will also need to be addressed.

Reply:
done

Reply:
Closing as resolved by: 13a2117